generators/app/templates/_.npmignore

```diff
@@ -0,0 +1 @@
+!dist/
```

Comment:
would this make npm only package up what is in `dist/`? If so, we probably should just include everything in the package instead.